QKMRZScanner/QKMRZScannerView.swift

```diff
@@ -282,7 +285,9 @@ extension QKMRZScannerView: AVCaptureVideoDataOutputSampleBufferDelegate {
                         let enlargedDocumentImage = self.enlargedDocumentImage(from: cgImage)
                         let scanResult = QKMRZScanResult(mrzResult: mrzResult, documentImage: enlargedDocumentImage)
                         self.delegate?.mrzScannerView(self, didFind: scanResult)
-                        AudioServicesPlaySystemSound(kSystemSoundID_Vibrate)
+                        if self.hapticsOnSuccess {
+                            AudioServicesPlaySystemSound(kSystemSoundID_Vibrate)
```

Comment:
Actually, this is just a regular vibration (works on older devices like iPhone 6 that don't have a taptic engine too), so it's not really a haptic feedback. I would suggest to rename the property to `vibrateOnResult` (for now).